solutions/src/main/java/ru/sberbank/school/task06/Test.java

```diff
@@ -0,0 +1,58 @@
+package ru.sberbank.school.task06;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Map;
+
+public class Test {
+    public static void main(String[] args) {
+        //Part1
+        Player p1 = new Player("Ivanov", "AkBars", 18);
+        Player p2 = new Player("Belov", "SKA", 27);
+        Player p3 = new Player("Kovalchuk", "SKA", 32);
+        Player p4 = new Player("Kaprizov", "CSKA", 20);
+
+        CountMap<Player> plMap = new CountMapImpl();
+        plMap.add(p1);
+        plMap.add(p2);
+        plMap.add(p3);
+        plMap.add(p4);
+        CountMap<Player> plMap2 = new CountMapImpl();
+        plMap2.addAll(plMap);
+        System.out.println(plMap2.size());
+        plMap2.remove(p2);
+        System.out.println(plMap2.size());
+        Map<Player, Integer> toMap = plMap.toMap();
+
+        // Part2
+        CollectionUtils<Integer> collectionUtils = new CollectionUtils<>();
+        List<Integer> list1 = new ArrayList<>();
+        List<Integer> list2 = new ArrayList<>();
```

Comment:
Distance between variable &#39;list2&#39; declaration and its first usage is 7, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1306037723>*